src/analyzer/analyzersilence.cpp

```diff
@@ -78,7 +78,8 @@ bool AnalyzerSilence::verifyFirstSound(
         mixxx::audio::FramePos firstSoundFrame) {
     const SINT firstSoundSample = findFirstSoundInChunk(samples);
     if (firstSoundSample < static_cast<SINT>(samples.size())) {
-        return mixxx::audio::FramePos::fromEngineSamplePos(firstSoundSample) == firstSoundFrame;
+        return mixxx::audio::FramePos::fromEngineSamplePos(firstSoundSample)
+                       .toLowerFrameBoundary() == firstSoundFrame;
```

Comment:
Should this also use the lower frame boundary of `firstSoundFrame`? This function cannot be sure that the caller passed in a frame position at a frame boundary.